apps/gimbtonomy/LEDMatrix.cpp

```diff
@@ -0,0 +1,47 @@
+/* LED Matrix Source File */
+
+#include "LEDMatrix.h"
+
+
+//Data constructor
+//LEDMatrix :: LEDMatrix (void)
+//{
+    //initialize PWM pins - pins tbd
+    
+    //redPWM(/*redLED*/);
+    //greenPWM(/*greenLED*/);
+    //bluePWM(/*blueLED*/);
+//}
+
+//Set how often it flashes/sec
+void LEDMatrix :: setFrequency(uint8_t msgFrequency)
+{
```

Comment:
Be aware of integer division, you will be setting everything to 0 by the way this is written rn